### PR TITLE
[Rec-IM] series edit, styling update, bug fixes

### PR DESCRIPTION
**Pull request by amos-cha**
Bug Fixes
--
- Activity Form's Post/Edit was broken due to mishandling of a Merge/overwritten in other PRs
- `Reduced` breadcrumb breaks on match page due to not ensuring that `Team[0]` or `Team[1]` is propagated resulting in an `[object Object]` call (hilarious, javascript is based)
- `new` Match Stats Form needs to be propagated in the back, however, CANNOT be propagated if there are no teams in the match
- Delete Match & Activity did not await `delete` thus instantly sending us back to a page that easily can route to a deleted match/activity

Series Edit
--
image
- Series Edit now has a `Team` Select tied to `Activity`'s teams so that teams can move in and out of the `Series`

Image
--
image
- Hover has reduced shadow
- "pops out"

image
- Minuscule shadow without hover.

Listing
--
Listings now have a minor inset box-shadow to emulate a light button press

image

image

Activity Page Update
--

- Edit button removed
- `Admin Settings` Dialog box replaced with `Menu`
- `Activity Edit` moved to `Admin Settings Menu`
image

- `Match Create` Button removed
- Location is now tied with Series (since matches can't be made without series to begin with)
image

Team Page Update
--
image
- Edit Button Removed
- Tasks done using settings button

Information Field Update (Form)
--
image
- Show selected items below (used when box becomes ellipsis)
- Select will take max grid space to ensure as much visibility as possible

### CODE REVIEW

File: src/views/RecIM/components/Forms/ActivityForm/index.jsx [hunk 1]
```diff
@@ -193,10 +183,17 @@ const ActivityForm = ({ activity, closeWithSnackbar, openActivityForm, setOpenAc
 
     let activityRequest = { ...currentInfo, ...newInfo };
 
+    activityRequest.statusID = activityStatusTypes.find(
```
Comment (silasw2) [inline src/views/RecIM/components/Forms/ActivityForm/index.jsx:186]: The statusID is not needed for the creation of an activity. It is fine to have it here but it is technically only needed if the activity is being modified.

File: src/views/RecIM/components/Forms/Form/index.jsx [hunk 2]
```diff
@@ -143,6 +140,25 @@ const Form = ({
     ));
   };
 
+  const DiaglogContent = () => {
```
Comment (silasw2) [inline src/views/RecIM/components/Forms/Form/index.jsx:143]:
```suggestion
  const DialogContent = () => {
```

File: src/views/RecIM/components/Forms/Form/index.jsx [hunk 3]
```diff
@@ -168,10 +184,7 @@ const Form = ({
       }}
       cancelButtonName={'cancel'}
     >
-      <ContentCard title={`${formTitles.name} Information`}>
-        {additionalContent}
-        {loading ? <GordonLoader /> : mapFieldsToInputs(fields)}
-      </ContentCard>
+      {DiaglogContent()}
```
Comment (silasw2) [inline src/views/RecIM/components/Forms/Form/index.jsx:187]:
```suggestion
      {DialogContent()}
```

File: src/views/RecIM/components/Forms/Form/index.jsx [hunk 4]
```diff
@@ -143,6 +140,25 @@ const Form = ({
...
+            formTitles.contentCardTitles
+              ? formTitles.contentCardTitles[index++]
+              : `${formTitles.name} Information`
+          }
+        >
+          {additionalContent}
+          {loading ? <GordonLoader /> : mapFieldsToInputs(set)}
+        </ContentCard>
+      );
+      return content;
```
Comment (silasw2) [inline src/views/RecIM/components/Forms/Form/index.jsx:158]:
```suggestion

```

File: src/views/RecIM/components/Forms/Form/index.jsx [hunk 5]
```diff
@@ -143,6 +140,25 @@ const Form = ({
     ));
   };
 
+  const DiaglogContent = () => {
+    let index = 0;
+    return fields.map((set) => {
+      let content = (
```
Comment (silasw2) [inline src/views/RecIM/components/Forms/Form/index.jsx:146]:
```suggestion
      return (
```

File: src/views/RecIM/components/Forms/ImageOptions/index.jsx [hunk 6]
```diff
@@ -83,7 +83,7 @@ const ImageOptions = ({
           IsLogoUpdate: true,
         };
 
-        editActivity(component.ID, activityRequest).then(() => {
+        await editActivity(component.ID, activityRequest).then(() => {
```
Comment (silasw2) [inline src/views/RecIM/components/Forms/ImageOptions/index.jsx:86]: I am not sure you understand what the await keyword does here. We are doing .then and .catch to handle whether the promise is fulfilled or not. Await is redundant at best here from my current understanding.

**Review by silasw2 — CHANGES_REQUESTED** [review]
Couple changes and comments. Will review when complete.

File: src/views/RecIM/components/Forms/ImageOptions/index.jsx (hunk 6 above)
Comment (amos-cha) [reply src/views/RecIM/components/Forms/ImageOptions/index.jsx]: Switched it, I don't have a clear memory of adding these awaits but it's a good catch

**Review by silasw2 — APPROVED** [review]
LGTM! Try to not do a develop merge and changes unrelated to making code work with develop merge in the future.

**Review by silasw2 — APPROVED** [review]
Sentiment has not changed. LGTM!

**Pull request merged**